src/StateManager.ts

```diff
@@ -37,7 +37,8 @@ export class StateManager {
 
   static clearFolderState(folder: string) {
     this.workspaceState.keys().forEach((key) => {
```

Comment:
now that i think about it, this method should probably not clear the auth0 user id as well